manifest.json

```diff
@@ -301,6 +301,13 @@
           "js": ["d3.min.js"]
         }
       },
+      "diff-match-patch": {
```

Comment:
Hi @seanlip,
I've added it this way PTAL, also I've given a  commit URL to download the file because they don't have a release URL.